src/app/component/layout/components/header/header.component.scss

```diff
@@ -169,7 +169,7 @@ $screen-320: " screen and (max-width : 320px)";
   cursor: pointer;
   width: auto;
   min-width: 138px;
-  padding: 0 4px 0;
+  padding: 15px 34px;
   height: 48px;
```

Comment:
Why these paddings are different?

Reply:
I fixed paddings according to mockup.
![image](https://user-images.githubusercontent.com/50843868/97700076-e518ad80-1ab3-11eb-901e-3ce9d4ea9a45.png)